### PR TITLE
feat: upgrade search to use the new search component

### PR DESCRIPTION
**Pull request by nunogois**
## About the changes
https://trello.com/c/hwCvqxr9

Upgrading the search field to use the new search seemed like an easy fix for all these issues.

### CODE REVIEW

**Review by FredrikOseberg — CHANGES_REQUESTED** [review]
Nice, but we might aswell place it similar to other search fields in the top right of the content container. Also, in this context is it a TableSearchField still, or should we rename it to something more general if we intend to use it for other usecases aswell?

File: src/component/common/Search/SearchSuggestions/SearchSuggestions.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    getColumnValues,
+    getFilterableColumns,
+    getFilterValues,
+    IGetSearchContextOutput,
+} from 'hooks/useSearch';
+import { useMemo, VFC } from 'react';
+import { SearchDescription } from './SearchDescription/SearchDescription';
+import { SearchInstructions } from './SearchInstructions/SearchInstructions';
+
+const randomIndex = (arr: any[]) => Math.floor(Math.random() * arr.length);
```
[the remark this answers is not in the document]
Comment (nunogois) [reply src/component/common/Search/SearchSuggestions/SearchSuggestions.tsx:14]: @sonatype-lift ignore

**Review by FredrikOseberg — APPROVED** [review]
Nice. Could we remove TableSearch now?

**Comment by nunogois** [comment]
> Nice. Could we remove TableSearch now?

Agreed - This is something that can be done in the Archive tables PRs. Otherwise I can create a later PR to try and clean everything up.

**Pull request merged**